confluent_kafka/src/confluent_kafka.h

```diff
@@ -231,6 +231,14 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 PyObject *c_parts_to_py (const rd_kafka_topic_partition_list_t *c_parts);
 rd_kafka_topic_partition_list_t *py_to_c_parts (PyObject *plist);
 
+PyObject * Kafka_generic_get_metadata(
+                Handle *self,
+                PyObject *args,
+                PyObject *kwargs);
+
+
+extern const char Kafka_generic_get_metadata_doc[];
```

Comment:
It actually needs to be char [] to compile because the initializer needs to be constant and something about copying the pointer is less constant than taking the address of the forward declared array.  I forget the details here, but I think it has something to do with the way relocations work when linking.